src/system_control/mod.rs

```diff
@@ -31,26 +32,41 @@ pub fn scb() -> SCB {
 }
 
 /// System Control Block
-#[derive(Copy, Clone)]
-pub struct SCB {
-    mem_addr: *const u32,
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+#[doc(hidden)]
+pub struct RawSCB {
+    unused: u32,
```

Comment:
The `SCB` has registers:
`CPUID`
`ICSR`
`32 bits reserved`
`AIRCR`
`SCR`
`CCR`
`32 bits reserved`
`SHPR2`
`SHPR3`

in that order in memory